.github/workflows/chart-update-readme-schema-golden-files.yaml

```diff
@@ -1,11 +1,11 @@
-name: "Chart - Update Golden Files and README"
+name: "Chart - Update Golden Files, Schema, and README"
 
 on:
   pull_request:
     branches:
       - main
     paths:
-      - ".github/workflows/chart-update-readme-and-golden-files.yaml"
+      - ".github/workflows/chart-update-readme-schema-golden-files.yaml"
```

Comment:
Let's just call it `chart chores` or something like that, so there's no need to rename the file and update the `name` every time we add something to it.